internal/watcher/config.go

```diff
@@ -33,7 +33,7 @@ type FilesystemConfig struct {
 	Inotify bool
 	Ignore  string
 
-	Pipeline         string
+	Pipeline         []string
```

Comment:
Not sure if because of [viper](https://github.com/spf13/viper) or [go-toml](https://github.com/pelletier/go-toml), but `[]string` takes both a TOML string and an array of strings, which fits perfectly with our use case.